src/condor_tests/CMakeLists.txt

```diff
@@ -306,7 +306,10 @@ endif ()
 if(LINUX)
 			condor_pl_test(test_ssh_sftp "Verify that condor_ssh_to_job can launch sftp" "quick;ctest" CTEST DEPENDS "src/condor_tests/ornithology;src/condor_tests/conftest.py")
 endif()
-
+if(LINUX OR WINDOWS)
+			# The curl library behaves differently on Mac.
```

Comment:
libcurl is one of the most heavily used open source libraries, works perfectly fine on Mac, and we continuously run a battery of tests across it.

If you see differences in test results on Mac, then it's far more likely you're (a) sensitive to more recent libcurl versions or (b) have test results indicating a real problem.

Extraordinary claims require extraordinary proof.  If we suspect there's a bug or misbehavior on Mac OS X, then let's link in the commit message to said bug and explain why we're susceptible to it.